Adding DLL Upload Experience and NetFxWorker upload

siddharth-ms opened the pull request.

preetriti1 commented:
Please add screenshots to where and how it will look in vscode

preetriti1 commented on apps/vs-code-designer/src/app/commands/deploy/deploy.ts:
```diff
@@ -180,6 +184,21 @@ async function deploy(
+ * @param fsPath publish path for logic app extension
+ */
+async function cleanupPublishBinPath(context: IActionContext, fsPath: string): Promise<void> {
+  const netFxWorkerBinPath = `${fsPath}\\bin\\NetFxWorker`;
```
Make sure these paths work in Linux as well

hartra344 commented:
Hey @siddharth-ms, per Pritis request please include screenshots of the VSCode experience or even better a video of it working. That makes it easier for us to review.

siddharth-ms commented:
> Hey @siddharth-ms, per Pritis request please include screenshots of the VSCode experience or even better a video of it working. That makes it easier for us to review.

Actually this is very similar to Artifact upload. I have added more detail in PR description. They just have to place the DLLs in the newly created directories and hit deploy. That's it

siddharth-ms commented:
> > > Hey @siddharth-ms, per Pritis request please include screenshots of the VSCode experience or even better a video of it working. That makes it easier for us to review.
> > 
> > 
> > Actually this is very similar to Artifact upload. I have added more detail in PR description. They just have to place the DLLs in the newly created directories and hit deploy. That's it
> 
> Can you please attach video of the experience. All we see if new folder structure that you added. We want to see what difference is this making for the user in vscode view and during deplo create a demo video of your feature, not code changes

Added to PR description.

ccastrotrejo commented on package-lock.json:
Please update your main branch and don't check the package-lock.json

ccastrotrejo commented on apps/vs-code-designer/src/app/commands/createNewProject/createNewProject.ts:
```diff
@@ -73,6 +73,7 @@ export async function createNewProjectInternal(context: IActionContext, options:
   await wizard.execute();
 
   await createArtifactsFolder(context as IFunctionWizardContext);
+  await createLibFolder(context as IFunctionWizardContext);
```
I wonder if this lib folder needs to be always here whenever the user creates a new project. With the demo It seems that its specific to a NuGet-based logic app. Can this lib folder be created when it is converted to NuGet based?
